crd/multitenancy/manifests/multitenancy.acn.azure.com_multitenantpodnetworkconfigs.yaml

```diff
@@ -71,8 +71,8 @@ spec:
                 description: Deprecated - use InterfaceInfos
                 type: string
               interfaceInfos:
-                description: InterfaceInfos describes all of the network container
-                  goal state for this Pod
+                description: InterfaceInfos describes all the network container goal
```

Comment:
revert this?

Reply:
Revert this will result in the CRD check fail.